wp-content/themes/workingnyc/views/components/collection.twig

```diff
@@ -23,12 +23,18 @@
         {% else %}
           {% if this.post.post_type == 'job-boards' and not is_home %}
           <div class="grid grid-cols-1 tablet:grid-cols-2 desktop:grid-cols-3 gap-3 auto-rows-[1fr]">
+          {% elseif this.post.post_type == 'employer-programs' %}
+          <div class="grid grid-cols-1 tablet:grid-cols-2 desktop:grid-cols-3 gap-3 auto-rows-[1fr]">
           {% else %}
           <div class="grid grid-cols-1 tablet:grid-cols-2 gap-3 auto-rows-[1fr]">
           {% endif %}
         {% endif %}
           {% for post in this.posts %}
+              {% if post.post_type == 'employer-programs' %}
+              <div class="max-w-full desktop:w-[400px] w-[343px] desktop:h-[286px] tablet:h-[261px] h-[245px]">
```

Comment:
- [ ] Don't set the widths, they will be set based on the browser size
- [ ] Don't set heights because the content might get cut off, only set min heights